package.json

```diff
@@ -40,5 +40,8 @@
     "start": "web-ext run --no-reload",
     "test": "echo 'Error: no test specified' && exit 0",
     "watch": "web-ext run"
+  },
+  "dependencies": {
+    "event-stream": "3.3.4"
```

Comment:
1. Should this be in `devDependencies` along w/ **npm-run-all**?
1. According to https://github.com/mysticatea/npm-run-all/issues/155, it looks like the issue was solved in **npm-run-all@4.1.5** by "switching from **ps-tree** package to **pidtree** package for security reasons".

Reply:
It looks like **nodemon** was fixed in [v1.18.7](https://github.com/remy/nodemon/releases/tag/v1.18.7) release. I'll comment in the **blurts-server** repo in case we want to bump that version and remove our pinned version of **event-stream**.

Reply:
Yay! Thanks!